woocommerce/woocommerce-ios
# [Woo POS] M2: Exit POS modal presentation

Closes: #13278


## Description


As a part of M2, info modal for exiting POS is added.
When user taps the three dot button in floating action control and taps "Exit POS" we present this info modal where user had the option to either exit POS or close the modal.
This way we are adding one more step of confirmation to make sure users to not exit POS by accident.

## Testing information


- Enter POS
- Tap three dot button in floating action control
- Check that the Exit POS modal is shown
- Tap on the X button and check that the modal is dismissed but you are still in POS
- Tap three dot button in floating action control again
- Tap on the "Exit" button and check that you have exited POS successfully

## Screenshots


![Simulator Screenshot - iPad Pro (11-inch) (4th generation) 17 2 - 2024-08-07 at 15 39 38](https://github.com/user-attachments/assets/13765c62-c398-469c-b3f6-84514cddb917)

https://github.com/user-attachments/assets/d29d5add-aa7a-48bf-bc7d-806b32596395

---
